src/models/room-member.js

```diff
@@ -45,6 +45,7 @@ import * as utils from "../utils";
  * @prop {string} membership The membership state for this room member e.g. 'join'.
  * @prop {Object} events The events describing this RoomMember.
  * @prop {MatrixEvent} events.member The m.room.member event for this RoomMember.
+ * @prop {boolean} disambiguate True if the client should disambiguate display names
```

Comment:
This is an indicator flag and not something that can be set external to the room member, so we should word it as such.
```suggestion
 * @prop {boolean} disambiguate True if the member's name is disambiguated.
```